pkg/controller/infrastructure/infraflow/ensure.go

```diff
@@ -163,12 +164,31 @@ func (fctx *FlowContext) ensureKubernetesRoutesCleanup(ctx context.Context) erro
 		return err
 	}
 
-	if err = infrastructure.DeleteRoutes(ctx, fctx.computeClient, cloudRoutes); err != nil {
-		// scale CCM back to originalScale in case if there are errors from DeleteRoutes() call
-		_ = scaleCCMto(originalScale)
-	}
+	for _, route := range cloudRoutes {
```

Comment:
This is a comment for the already existing function name. Since we also have a function for the delete called `ensureKubernetesRoutesDeleted`, I would rename this function that is specific to the ipv6 migration.